src/GlobalHeader/index.js

```diff
@@ -418,7 +413,7 @@ const GlobalHeader = ({ globalNav, pages, docs, location, ...props }) => {
             )}
           </View>
           <View gridArea='profile' justifySelf='center'>
-            {globalNav?.signIn && (
+            {globalNav.signIn && (
```

Comment:
Same as above.

Reply:
yes let me know what works best for you.